fastplotlib/utils/generate_add_methods.py

```diff
@@ -0,0 +1,38 @@
+import inspect
+import fastplotlib
+from fastplotlib.graphics import *
+import sys
+
+modules = dict(inspect.getmembers(fastplotlib.graphics))['__all__']
+
+
+def generate_add_graphics_methods():
+    # clear file and regenerate from scratch
+    open('add_graphics.py', 'w').close()
+
+    sys.stdout = open('add_graphics.py', 'w')
```

Comment:
just use regular file open and writing instead of stdout and print, I didn't know this is even possible :laughing: